src/stylesheets/project_specific/developer/_forms.scss

```diff
@@ -1,5 +1,85 @@
 
-  /* Braintree-specific class */
-  .field_with_errors {
-    @extend .has-error;
-  }
+/* Braintree-specific class */
+.field_with_errors {
+  @extend .has-error;
+}
+
+.dollah-sign {
+  display: inline-block;
+  float: left;
+  margin-right: 6px;
+  font-size: 24px;
+  line-height: 40px; // to match input field height
+}
+
+input[disabled],
+input[disabled] + .dollah-sign {
+  color: #ccc;
+}
+
+// Toggle Switch
+// Should we move to common/_forms partial?
```

Comment:
Yep.  It's available to all developer forms right now.  I want to check with Ekta, though, before deciding that this should be more widely available.